std/special/compiler_rt/extendXfYf2.zig

```diff
@@ -11,12 +11,12 @@ pub extern fn __extendsftf2(a: f32) f128 {
 }
 
 pub extern fn __extendhfsf2(a: u16) f32 {
-    return extendXfYf2(f32, f16, @bitCast(f16, a));
+    return extendXfYf2(f32, f16, a);
 }
 
 const CHAR_BIT = 8;
 
-inline fn extendXfYf2(comptime dst_t: type, comptime src_t: type, a: src_t) dst_t {
+inline fn extendXfYf2(comptime dst_t: type, comptime src_t: type, a: var) dst_t {
```

Comment:
Can I request that we always pass the bitcasted integer type here? e.g.  `a: @IntType(false, @typeInfo(src_t).Float.bits)` rather than `a: var` (and then update the callsites).

I think this is easier to reason about than using a generic function.

Reply:
More type-safety is always welcome!